src/effects/RealtimeEffectList.h

```diff
@@ -34,11 +35,14 @@ class RealtimeEffectList final
    RealtimeEffectList &operator=(const RealtimeEffectList &) = delete;
 
 public:
+   using Lock = spinlock;
    using States = std::vector<std::shared_ptr<RealtimeEffectState>>;
 
    RealtimeEffectList();
    virtual ~RealtimeEffectList();
 
+   Lock &GetLock() const { return mLock; }
```

Comment:
As I can see `RealtimeEffectList` interface does not provide any useful const methods, so locking for reading isn't the case.